db.rb

```diff
@@ -1,9 +1,49 @@
 require "active_record"
 
-ActiveRecord::Base.establish_connection(
-  adapter: "sqlite3",
-  database: "db/db.sqlite3"
-)
+def setup_db(db_location)
+  ActiveRecord::Base.establish_connection(
+    adapter: "sqlite3",
+    database: db_location
+  )
+end
+
+def wipe_db
```

Comment:
I like the idea of abstracting all the DB clearing for all the tests, but I'm gonna suggest that you just clear what you need in all of the test files instead of having a "wipe all" method. Or just delete the db file and re-migrate it.

Reply:
So I think I may actually need your help with this. I've done some research on ActiveRecord's while doing this refactor, but I'm still not completely clear on how migration or our db setup work. I do think we need a separate DB for testing (if only so that you wouldn't need to delete your production DB just to test), but I'm not sure if doing it how I did (just copying our current db and then modifying where we're reading from) is the best way.

Is there some better way to use our setup to create and keep up-to-date another db for testing? I'd imagine it'd be some work in the Rakefile, but I'm honestly clueless there.